src/pages/graphql/schema/customer/mutations/exchange-external-customer-token.md

```diff
@@ -0,0 +1,90 @@
+---
+title: exchangeExternalCustomerToken mutation
+---
+
+# exchangeExternalCustomerToken mutation
```

Comment:
This topic is missing information, but I don't know what exactly. I don't expect a full discussion of OAuth 1.0, but more context is needed. 

You originally had the four Integration fields listed under a section called "Authentication headers" (or something like that), but I removed that section, because it didn't make sense to me. If it does belong, then you need an example, and perhaps you need to update the [Headers](https://developer.adobe.com/commerce/webapi/graphql/usage/headers/) topic.

I suspect technical information also needs to be added to the storefront docs, maybe near, the [user auth drop in](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/user-auth/) docs.